tests/mapping/test_mapping_script.py

```diff
@@ -0,0 +1,121 @@
+import unittest
+import rdflib
+import datetime
+import os
+import yaml
+from pathlib import Path
+
+from lebedigital.mapping.emodul_mapping import generate_knowledge_graph, get_date_time_value
+
+class TestMappingScript(unittest.TestCase):
+
+    def query_and_test_result(self, typeOfQuery, predicate, prop, g):
+        print(f"""Query for {typeOfQuery}:""")
+        q = f"""
+                SELECT ?o WHERE {{
+                        {predicate} {prop} ?o .
+                }}
+                """
+        print(q)
+        result = g.query(q)
+        return result
+
+
+    def print_next(self):
+        print("Pass")
+        print("###################Next Query#####################################")
+
+
+    def print_first(self):
+        print("###################First Query####################################")
+
+
+    def test_generate_knowledge_graph(self):
+        metadataPath = 'testMetaData.yaml'
+        filename = "knowledgeGraph.ttl"
+        print("Start testing:")
+        # Generate a testMetadata file
+        print("Generate metadata file for test:")
+        target_data = {'experimentName': 'BA-Losert MI E-Modul 28d v. 04.08.14 Probe 4',
+                       'software_specification': 'MTS793|MPT|DEU|1|2|,|.|:|49|1|1|A',
+                       'operator_timestamp': '13:25:39',
+                       'operator_date': '01.09.2014',
+                       'tester_name': 'Kh',
+                       'specimen_name': 'BA-Losert E-Modul 28d v. 04.08.14 Probe 4',
+                       'remark': 'Kraftgeregelt 3,9 kN/s',
+                       'weight': 5342.0,
+                       'weight_unit': 'g',
+                       'diameter': 98.6,
+                       'length': 300.3,
+                       'length_unit': 'mm',
+                       'mix_file': '2014_08_05 Rezeptur_MI.xlsx'}
+
+        with open(metadataPath, 'w') as yamlFile:
```

Comment:
are these newly created files added to the gitignore

Reply:
These files (`testMetadata.yaml`, `knowledgeGraph.ttl` will be removed at the end of the script and  no track of them can be found in the end. So there is no need to add them to gitignore.